config/webpack/index.js

```diff
@@ -1,4 +1,5 @@
+const path = require('path');
 const env = process.env.ENV || 'dev';
 const config = require(`./webpack-${env}`);
```

Comment:
yes I guess so, it's more clean to use that instead of '.'
I really just copied the changes from commits you made to cornerstone and cornerstoneTools and checked that I can build in windows.